README.md

```diff
@@ -231,6 +231,20 @@ status = "generate"
 Note that you must not place `Gtk.*` into the `generate` array and
 additionally configure its members.
 
+You can control the generation of constants in a similar fashion:
+
+```
+[[object]]
+name = "Gtk.*"
+status = "generate"
+    [[object.constant]]
+    pattern = "*"
+    # No constants will be generated
+    ignore = true
```

Comment:
Can the pattern also contain something useful and then only those constants that match the pattern will be ignored?

Reply:
yes, I tried, you can use `name` as well